docker-compose.yml

```
@@ -46,6 +46,15 @@ services:
       - postgres_data_test:/var/lib/postgresql/data/
     env_file:
       - ./.env
+  pgadmin:
+    container_name: pgadmin4_container
+    image: dpage/pgadmin4
+    restart: always
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@admin.com
+      PGADMIN_DEFAULT_PASSWORD: root
+    ports:
+      - "5050:80"
 
```

should the environment variables be in the `.env` file instead of the docker compose file?
```suggestion
      - "5050:80"
    env_file:
      -  ./.env
    depends_on:
      - db
```